python/tests/expressions/test_visitors.py

```diff
@@ -853,19 +868,11 @@ def test_manifest_evaluator_less_than_no_overlap():
             upper_bound=_to_byte_buffer(StringType(), "b"),
         )
     )
+    assert _ManifestEvalVisitor(string_schema, expr).eval(manifest)
 
-    assert _create_manifest_evaluator(expr).eval(manifest)
```

Comment:
Is the `_create_manifest_evaluator` method used anywhere? Why keep it?

Reply:
Removed